src/workloads/query/TenMDocCollection_IntId.yml

```diff
@@ -62,7 +73,22 @@ ActorTemplates:
         Operations:
         - OperationName: find
           OperationCommand:
-            Filter: {$or: {^FixedGeneratedValue: {fromGenerator: {^Array: {distinct: true, number: 100000, of: {_id: {^RandomInt: {min: 0, max: 9999999}}}}}}}}
+            Filter: {$or: [{_id: {^RandomInt: {min: 0, max: 9999999}}}]}
+- TemplateName: IDFieldPointProject
+  Config:
+    LoadConfig:
+      Path: TenMDocCollection_Base.tmpl
+      Key: ActorTemplateConfig
+      Parameters:
+        MaxPhases: *MaxPhases
+        Operations:
+        - OperationName: aggregate
+          OperationCommand:
+            Pipeline:
+            - $match:
+                id: {^RandomInt: {min: 0, max: 9999999}}
+            - $project:
+                id: '$id'
```

Comment:
I think you should use '1' or 'true' otherwise this won't be considered a "simple" projection: https://github.com/mongodb/mongo/blob/2ef28165da8a2deb91bc46757aec32c0f419ae81/src/mongo/db/query/projection.h#L143